### PR TITLE
Update benchmark scripts  code with pylint check 

### PR DESCRIPTION
**Pull request by jingkl**
Signed-off-by: zilliz <zilliz@zillizdeMacBook-Pro.local>

- [ ] A reference to a related issue in your repository.
  
  Each PR is related to an issue, and you need to list that issue.

- [ ] A description of the changes proposed in the pull request.

  A brief introduction to this PR.

- [ ] Add delight to the experience when all tasks are complete :tada:

### CODE REVIEW

File: benchmark_test/scripts/milvus_helpers.py [hunk 1]
```diff
@@ -1,164 +1,169 @@
...
             if not self.has_collection(collection_name):
-                field1 = FieldSchema(name="id", dtype=DataType.INT64, descrition="int64", is_primary=True,
-                                     auto_id=False)
-                field2 = FieldSchema(name="embedding", dtype=DataType.FLOAT_VECTOR, descrition="float vector",
-                                     dim=VECTOR_DIMENSION, is_primary=False)
-                schema = CollectionSchema(fields=[field1, field2], description="collection description")
-                self.collection = Collection(name=collection_name, schema=schema, shards_num=SHARDS_NUM)
-                LOGGER.debug("Create Milvus collection: {}".format(self.collection))
-                return "OK"
+                field1 = FieldSchema(name="id", dtype=DataType.INT64, descrition="int64", is_primary=True,auto_id=True)
```
Comment (shiyu22) [inline benchmark_test/scripts/milvus_helpers.py:46]: I think `auto_id=False` before.

File: benchmark_test/scripts/milvus_helpers.py [hunk 2]
```diff
@@ -1,164 +1,169 @@
...
+                field2 = FieldSchema(name="embedding", dtype=DataType.FLOAT_VECTOR, descrition="float vector", dim=VECTOR_DIMENSION, is_primary=False)
+                schema = CollectionSchema(fields=[ field1,field2], description="collection description")
+                self.collection = Collection(name=collection_name, schema=schema)
+                LOGGER.debug(f"Create Milvus collection: {self.collectio}")
```
Comment (shiyu22) [inline benchmark_test/scripts/milvus_helpers.py:50]: Modify: LOGGER.debug(f"Create Milvus collection: {self.collection}")

File: benchmark_test/scripts/performance_test.py [hunk 3]
```diff
@@ -43,11 +42,11 @@ def performance(client, collection_name, search_param):
     result_filename = collection_name + '_' + str(search_param) + '_performance.csv'
     performance_file = os.path.join(PERFORMANCE_RESULTS_PATH, result_filename)
 
-    with open(performance_file, 'w+') as f:
+    with open(performance_file, 'w+',encoding='utf-8') as f:
```
[the remark this answers is not in the document]
Comment (jingkl) [reply benchmark_test/scripts/performance_test.py]: Solved

File: benchmark_test/scripts/milvus_helpers.py [hunk 4]
```diff
@@ -1,164 +1,169 @@
...
+                self.set_collection(collection_name)
+            return "OK"
         except Exception as e:
-            LOGGER.error("Failed to load data to Milvus: {}".format(e))
+            LOGGER.error(f"Failed to load data to Milvus: {e}")
             sys.exit(1)
 
-    # Batch insert vectors to milvus collection
-    def insert(self, collection_name, vectors, ids=None):
+    def insert(self, collection_name, vectors):
```
[the remark this answers is not in the document]
Comment (jingkl) [reply benchmark_test/scripts/milvus_helpers.py:61]: Solved

File: benchmark_test/scripts/milvus_helpers.py [hunk 5]
```diff
@@ -1,164 +1,169 @@
...
+            num =self.collection.num_entities
+            LOGGER.debug(f"Successfully get the num:{num} of the collection:{collection_name}")
             return num
         except Exception as e:
-            LOGGER.error("Failed to count vectors in Milvus: {}".format(e))
+            LOGGER.error(f"Failed to count vectors in Milvus: {e}")
             sys.exit(1)
 
-    # create a partition for Milvus
+    def get_index_params(self, collection_name):
```
[the remark this answers is not in the document]
Comment (jingkl) [reply benchmark_test/scripts/milvus_helpers.py:120]: Solved

**Review by shiyu22 — APPROVED** [review]

**Pull request merged**